cryptoshred-core/src/main/java/eu/prismacapacity/cryptoshred/core/CryptoContainer.java

```diff
@@ -129,25 +125,25 @@ protected CryptoContainer(@NonNull Class<T> class1, @NonNull CryptoAlgorithm alg
 	}
 
 	@Getter
-	private Class<?> type;
+	private final Class<?> type;
```

Comment:
not sure if this is semantically correct on a serialized/deserialized object

Reply:
if in doubt I'd change it back then
```suggestion
	private Class<?> type;
```